tsdb/head.go

```diff
@@ -1821,7 +2009,12 @@ func (s *memSeries) append(t int64, v float64, appendID uint64) (success, chunkC
 	c := s.head()
 
 	if c == nil {
-		c = s.cut(t)
+		// Out of order sample. Sample timestamp is already in the mmaped chunks, so ignore it.
```

Comment:
Let's put this comment inside this if - otherwise I was reading this as c== nil means out of order.

Reply:
Hah sorry for that! moving inside.